### PR TITLE
update Dockerfile syntax

### PR DESCRIPTION
**Pull request by Leticia-maria**
I have corrected the syntax of the Dockerfile following the best practices rules of Dokcer documentation

### CODE REVIEW

**Comment by fredrikekre** [comment]
What best practice are you referring to? Note that the change here is invalid, all those packages are arguments to `apt get` and not their own commands.

**Comment by Leticia-maria** [comment]
> What best practice are you referring to? Note that the change here is invalid, all those packages are arguments to `apt-get` and not their own commands.

Yes, I have corrected the apt-get error. But the good practice that I have mentioned is here:  [https://docs.docker.com/develop/develop-images/dockerfile_best-practices/](url), [https://docs.docker.com/develop/develop-images/dockerfile_best-practices/](url) and [https://docs.docker.com/develop/develop-images/dockerfile_best-practices/](url)

**Comment by guilhermebodin** [comment]
Shouldn`t it be like this https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments ? Without many apt-get commands and sorted alphanumerically

**Comment by Leticia-maria** [comment]
> Shouldn`t it be like this https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments ? Without many apt-get commands and sorted alphanumerically
> 
> 

Yes, I have corrected this😊

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,14 @@
 FROM julia:latest
 
-RUN apt-get update && apt-get install -y build-essential libatomic1 python gfortran perl wget m4 cmake pkg-config git
+RUN apt-get update \
+    install -y \
```
Comment (KristofferC) [inline .devcontainer/Dockerfile:4]: This should be on the same line as `apt-get`?

Comment (fredrikekre) [reply .devcontainer/Dockerfile:4]: And also need `apt-get` before `install`. The only thing "wrong" with the file from the start is perhaps that they recommend sorting the packages (which isn't even done). The last line should not end with `\` either. As a general rule it is good to try out changes you make before submitting pull request and having people spend time reviewing.

**Pull request closed without merging**